docs/Documentation/Drivers/Classic/GlobalParam.md

```diff
@@ -197,25 +208,25 @@ _Examples._ a standard four column daily forcing data file will be defined as:
+| ROOT_ZONES         | integer         | N/A                 | Number of defined root zones defined for root distribution.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| VEGPARAM_ALB       | string          | TRUE or FALSE       | If TRUE the vegetation parameter file contains an extra line for each vegetation type that defines monthly ALBEDO values for each vegetation type for each grid cell. Default = FALSE.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| ALB_SRC            | string          | N/A                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| ALBEDO_SRC         | string          | N/A                 | This option tells VIC where to look for ALBEDO values:FROM_VEGLIB = Use the ALBEDO values listed in the vegetation library file.FROM_VEGPARAM = Use the ALBEDO values listed in the vegetation parameter file. Note: for this to work, VEGPARAM_ALB must be TRUE.FROM_VEGHIST = Use the ALBEDO values listed in the veg_hist forcing files. Note: for this to work, ALBEDO must be supplied in the veg_hist files and listd in the global parameter file as one of the variables in the files. Default = FROM_VEGLIB.                                                                                                                                                     |
```

Comment:
There is no `ALBEDO_SRC` in the code. The description should be re-assigned to `ALB_SRC` and the `ALBEDO_SRC` entry can be deleted. Also requires that the following two files are updated:

./tests/science/global_param.classic.4.2_ecflux.txt:# ALBEDO_SRC      FROM_VEGPARAM
./tests/science/global_param.classic.5.0_ecflux.txt:# ALBEDO_SRC            FROM_VEGPARAM